README.md

```diff
@@ -49,7 +49,7 @@ In this project, I connected my React & Redux application to an existing API to
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-[ Demo is not available for this stage of progress]()
+([ https://shimmering-kangaroo-bc7705.netlify.app/Books])
```

Comment:
[OPTIONAL] Your netlify link looks broken. Kindly fix that. You can follow this structure for the markdown. ```[Bookstore](netlifylink)```

Reply:
Thanks, Fixed :)